gemini_test.py

```diff
@@ -138,6 +139,33 @@ def get_email_data(self):
             self.log.warning("Failed to get gemini version as loader instance is not created")
             gemini_version = ""
 
+        try:
+            seed = self.params.get("gemini_seed")
+            if not seed:
+                seed_match = re.search(r"--seed (?P<seed>\d+) ", self.gemini_results["cmd"])
+                if seed_match:
+                    seed = seed_match.groupdict().get("seed")
+                else:
+                    seed = -1
+
+            # pylint: disable=no-member
+            self.test_config.argus_client().submit_gemini_results({
+                "gemini_command": self.gemini_results["cmd"],
+                "gemini_read_errors": self.gemini_results["results"].get("read_errors", -1),
+                "gemini_read_ops": self.gemini_results["results"].get("read_ops", -1),
+                "gemini_seed": seed,
+                "gemini_status": self.gemini_results["status"],
+                "gemini_version": gemini_version,
+                "gemini_write_errors": self.gemini_results["results"].get("write_errors", -1),
```

Comment:
I was thinking results are a list, does it work this way?

Reply:
I've looked into `results_gemini.html` and it appears to be a `dict` as template calls `.items()` on it.

Reply:
@k0machi 

```
{% for res in results %}
```
there a loop on the template, I think @soyacz is correct.

Reply:
Yep, didn't quite notice it, it's a list of dicts